features/FxA-54-push-on-status-change/README.md

```diff
@@ -0,0 +1,84 @@
+# Push Notifications for Account Status Changes
+
+https://mozilla.aha.io/features/FXA-54
+
+## Stories
+
+As a user signed in to an FxA connected device,
+I want my device to update its UI in a timely manner
+when I interact with my account from other devices.
+
+As a developer of client code for a connected device,
+I want to easily receive notifications so that I can
+respond to account status changes without resorting to
+polling the server.
+
+As a developer of Firefox Accounts,
+I want to stop Firefox from polling for account verification
+because this is the single biggest source
+of system load in production.
+
+## Success Criteria
+
+This feature will be successful if
+we successfully eliminate polling for account verification
+from the browser code on Desktop and Fennec.
+
+As a concrete metric
+we can measure the time between account verification
+and the use of a keyFetchToken by the browser,
+an action that it will only perform
+when it knows the account has been verified.
```

Comment:
Would it be simpler to use the fxa-auth-server load % on verification endpoints? Can we use some metric just from fxa-auth-server as a concrete metric instead?